src/pybind/mgr/rook/module.py

```diff
@@ -415,6 +415,8 @@ def _service_add_decorate(self, typename, spec, func):
 
     def add_nfs(self, spec):
         # type: (NFSServiceSpec) -> RookCompletion
+        if spec.placement.count == None:
```

Comment:
The error in the tracker is 
```
Invalid value: "null": spec.server.active in body must be of type integer
```
Should this instead be setting `spec.server.active`?

Reply:
Here `spec.server.active` is initialized with `spec.placement.count` value.

https://github.com/ceph/ceph/blob/master/src/pybind/mgr/rook/rook_cluster.py#L469-L474

Reply:
I see, this is a different object.